include/aligator/modelling/costs/sum-of-costs.hxx

```diff
@@ -25,12 +25,19 @@ CostStackTpl<Scalar>::CostStackTpl(xyz::polymorphic<Manifold> space,
         ALIGATOR_RUNTIME_ERROR(msg);
       }
     }
+
+    for (std::size_t i = 0; i < comps.size(); i++) {
+      components_.emplace(
+          std::make_pair(i, std::make_pair(comps[i], weights[i])));
```

Comment:
Why do you emplace a std::pair here? Can't you just do `components_.emplace(i, std::make_pair(comps[i], weights[i])));`?

Reply:
Nice catch, I think I was debugging and left it there